specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerActiveConfigurationList.json

```diff
@@ -12,35 +12,93 @@
         "value": [
           {
             "configType": "Connectivity",
```

Comment:
please correct me if I am wrong: I thought we change to CommitType?

Reply:
same Q for "configType": "UserPolicy",

Reply:
Since we will have userconfig and adminconfig, so I think we should use configType back, and remove CommitType